olp-cpp-sdk-dataservice-read/src/repositories/PartitionsRepository.h

```diff
@@ -60,6 +61,12 @@ class PartitionsRepository {
   static model::Partition PartitionFromSubQuad(
       const model::SubQuad& sub_quad, const std::string& partition);
 
+  static PartitionsResponse GetAggregatedPartitionForVersionedTile(
+      client::HRN catalog, std::string layer,
```

Comment:
```suggestion
      const client::HRN& catalog, const std::string& layer,
```
Any async operations? Then make them pass by reference.